### ISSUE TITLE
https://github.com/w3c/WebID/blob/main/spec/README

### ISSUE THREAD

**Issue by webr3** [w3c/WebID#18]
```
Source
------

You can read, branch and modify the source code for this specification via
github:

https://github.com/webid-community/webid-spec

Feedback
--------

Don't e-mail patches to the editors, don't send tweets, IMs, or e-mails.
Log bugs if you want to request changes to the spec, it is the only way
you can make sure that your input will be tracked and considered by
the group:

https://github.com/webid-community/webid-spec/issues
```

`/webid-community/webid-spec` hasn't had any action in 12 years, is the README incorrect?

**Comment by bblfish** [comment]
I think the https://github.com/webid-community/ repo was a copy of the mercurial repository to GitHub. We were using mercurial at the time.

It seems that there is a version in this repository here [spec/README](https://github.com/w3c/WebID/blob/main/spec/README). 

I would be all for moving that to the root of the GitHub repo here and adapting the language.

**Comment by csarven** [comment]
AFAIK/CT, that's from the old landing page for WebID right? As for the Identity spec, I have taken webid-community/webid-spec and dvcs.w3.org repo for the WebID identity spec into account. See the "Aside" in https://github.com/w3c/WebID/issues/5#issue-1153260476 . To the best of my and others' abilities, I believe what's in this repo is latest and most accurate / also most closely aligned with what was published at https://www.w3.org/2005/Incubator/webid/spec/identity/ .

Resolving issue 5 should set things straight. Ditto a new publication (whether from WebID CG or Solid WG, if/when it is active.)

**Comment by webr3** [comment]
> It seems that there is a version in this repository here [spec/README](https://github.com/w3c/WebID/blob/main/spec/README).
> 
> I would be all for moving that to the root of the GitHub repo here and adapting the language.

Yes, please update, else one cannot follow instructions to provide pull requests

**Comment by bblfish** [comment]
there is a PR for that now. I did not move the README as I am not sure if that would be right.
